packages/fuse/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "fuse"
+version = "2.9.9"
```

Comment:
According to the GitHub repo, this is a maintance version, and folks should move to the latest 3.x version. I didn't see any requirements on the version for fuse in SOCI, but it will be best to ask the developers to make sure we meet SOCI's requirements.